src/Client.php

```diff
@@ -183,9 +176,14 @@ protected function getApiContentType()
      */
     public function getAsset($assetId, $locale = null)
     {
-        $locale = null === $locale ? $this->defaultLocale : $locale;
+        $locale = $locale ?: $this->defaultLocale;
 
-        return $this->requestAndBuild('GET', 'assets/'.$assetId, [
+        $instanceId = $assetId.'-'.($locale ?: '*');
```

Comment:
Should this be abstracted into a `getCacheKey` method?

Reply:
Possibly, yeah 🤔

Reply:
But then I would need to supply the locales info to the `getCacheKey` method...